tmva/sofie/inc/TMVA/ROperator_LayerNormalization.hxx

```diff
@@ -169,18 +169,20 @@ public:
 
       std::stringstream out;
 
-      out << "//---- Layer Normalization  operator " << OpName << "\n";
+      out << "//---- Layer Normalization  operator " << opName << "\n";
 
       // Loop over all the normalized axes i.e. [axis, ..., size)
-      out << SP << "std::vector<size_t> " << OpName << "_InputShape ({";
+      std::vector<std::string> inputShape(fSize);
+      //out << SP << "std::vector<size_t> " << opName << "_InputShape ({";
       for (size_t i = 0; i < fSize; i++) {
-         out << fShapeX[i].GetVal();
-         if (i + 1 < fSize) {
-            out << ",";
-         }
+         inputShape[i] = fShapeX[i].GetVal();
+         //out << fShapeX[i].GetVal();
+         //if (i + 1 < fSize) {
+         //   out << ",";
+         //}
       }
-      out << "});\n";
-      std::string inputShape = OpName + "_InputShape";
+      //out << "});\n";
+      //std::string inputShape = opName + "_InputShape";
```

Comment:
we can maybe remove the commented code here

Reply:
Done